channel/persistence/test/peerchans.go

```diff
@@ -73,10 +74,17 @@ func (pc peerChans) Delete(id channel.ID) {
 	}
 }
 
-func peerKey(a wire.Address) string { return string(a.Bytes()) }
+func peerKey(a wire.Address) string {
```

Comment:
Why not use binary marshaller?

Reply:
See https://github.com/hyperledger-labs/go-perun/pull/258#discussion_r757694759.